stable_baselines/acktr/acktr.py

```diff
@@ -40,15 +37,18 @@ class ACKTR(ActorCriticRLModel):
     :param tensorboard_log: (str) the log location for tensorboard (if None, no logging)
     :param _init_setup_model: (bool) Whether or not to build the network at the creation of the instance
     :param async_eigen_decomp: (bool) Use async eigen decomposition
+    :param kfac_update: (int) update kfac after kfac_update steps
     :param policy_kwargs: (dict) additional arguments to be passed to the policy on creation
+    :param gae_lambda: (float) Factor for trade-off of bias vs variance for Generalized Advantage Estimator
+        If negative (default), then the classic advantage will be used instead of GAE
```

Comment:
This could be `None` for no GAE, similar to other "None if you want it disabled" parameters. I got confused at the first sight of negative GAE \lambda and thought I have missed something crucial.

Reply:
good point ;)